third_party/libusb/naclport/src/libusb_over_chrome_usb.cc

```diff
@@ -523,6 +533,17 @@ int LibusbOverChromeUsb::LibusbOpen(
   if (!result.is_successful()) {
     GOOGLE_SMART_CARD_LOG_WARNING << "LibusbOverChromeUsb::LibusbOpen " <<
         "request failed: " << result.error_message();
+    // modify the devices (fake) bus number that we report so that PCSC will 
+    // retry to connect to the device once it updates the device list
+    if (kBusNumbers.find(dev->chrome_usb_device().device) != 
+        kBusNumbers.end()) {
+      kBusNumbers[dev->chrome_usb_device().device] = std::min(
+          static_cast<uint8_t>(
+          kBusNumbers[dev->chrome_usb_device().device] + 1),
+          kMaximumReconnectAttempts);
+    } else {
+      kBusNumbers[dev->chrome_usb_device().device] = kDefaultBusNumber + 1;
+    }
```

Comment:
Just thinking out loud: Probably we could extract this custom logic out of the LibusbOverChromeUsb class, so that the latter would behave in a standard way by default, and only a separate "hook" class would be performing our PC/SC-Lite specific logic.

This would be nice to do, but I guess we can skip this if this makes the things simpler.

The main benefit would be that removing these hacks would allow to directly reuse our libusb port in other projects. But I don't know if anyone uses it today, so we can postpone this consideration till later.